posydon/interpolation/IF_interpolation.py

```diff
@@ -1140,8 +1140,10 @@ def predict(self, Xt):
         if np.any(wnan):
             Ypred[wnan, :] = self.interpolator[1].predict(Xt[wnan, :])
         if np.any(wnan):
+            dists, _ = self.interpolator[1].kneighbors(Xt[wnan, :])
```

Comment:
Wouldn't it be saver to specify the number of neighbors here, just to ensure that we are independent from the global settings? Otherwise we may check for too many neighbors, which are further away.
Additionally, I'm not sure whether this works as we'd need it: we are interested in the distance on each axis individually because they are probably scaled independently.